modules/dop/providers/projectpipeline/service_impl.go

```diff
@@ -811,7 +844,13 @@ func (p *ProjectPipelineService) failRerunOrRerunPipeline(rerun bool, pipelineDe
 	if err != nil {
 		return nil, apiError.InternalError(err)
 	}
-	return dto, nil
+
+	pipeline, err := pipelineDTOToStructPb(dto)
```

Comment:
convert to structpb.Value, why?

Reply:
PipelineDTO is not provided, so use the universal type

This pb should not be submitted in this pr